airflow/ui/src/queries/useClearTaskInstances.ts

```diff
@@ -60,13 +60,21 @@ export const useClearTaskInstances = ({
       onSuccessDryRun(data);
     } else {
       const taskInstanceKeys = (variables.requestBody.task_ids ?? [])
+        .filter(
+          (taskId): taskId is string | [string, number] =>
```

Comment:
I don't think we should use `is` keyword. That's only for Type Guard I don't think we should need that.